spec/controllers/kudos_controller_spec.rb

```diff
@@ -16,6 +16,7 @@
           request.headers["HTTP_REFERER"] = referer
           post :create, params: { kudo: { commentable_id: work.id, commentable_type: "Work" } }
           it_redirects_to_with_comment_notice(referer, "Thank you for leaving kudos!")
+          expect(assigns(:kudo).user).to be_nil
```

Comment:
I'm going to be a pain in the ass and suggest

a) we should wrap these tests in a `context "when kudos giver is a guest"`
b) we update the descriptions in all the new/modified specs to mention we're checking that `user` is/is not assigned correctly, not just checking the the `referer` and notice. (If I wanted to be an even bigger pain in the ass, I'd say we probably shouldn't be checking two things at once like that, but I don't care that strongly.)